__tests__/api/check-status.test.ts

```diff
@@ -0,0 +1,52 @@
+import handler from '../../pages/api/check-status'
+import { createMocks, createRequest, createResponse } from 'node-mocks-http'
+import { CheckStatusRequest } from '../../lib/StatusTypes'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+/**
+ * NextApiRequest and node-mocks-http createRequest types union
+ * @see: https://github.com/howardabrams/node-mocks-http/issues/255#issuecomment-1136674043
+ */
+export type ApiRequest = NextApiRequest & ReturnType<typeof createRequest>
```

Comment:
don't need to export this, should be local to the test (at least while in this file)

Reply:
Good point, forgot to remove the export.